Makefile

```diff
@@ -43,15 +34,45 @@ all: deps
 deps:
 	cd $(SWIFTNAV_ROOT)/tasks && bash setup.sh && cd $(SWIFTNAV_ROOT)
 
+.conda_py27:
+	conda create -p $(PWD)/.conda_py27 python=2.7 --yes
+
+.conda_py35:
+	conda create -p $(PWD)/.conda_py35 python=3.5 --yes
+
+tox_all:
+	@echo Using TESTENV=$(TESTENV), TOXENV=$(TOXENV)...
+	tox $(if $(filter y,$(VERBOSE)), -v,)
+
+tox_Darwin: export TESTENV:=$(TESTENV)
+tox_Darwin: export TOXENV:=$(TOXENV)
+tox_Darwin: tox_all
+
+tox: .conda_py27 .conda_py35
+tox: export PATH:=$(CURDIR)/.conda_py35/bin:$(CURDIR).conda_py27/bin:$(PATH)
```

Comment:
would the path expansions be neater in .conda_py27 and ._conda_py35 targets?

Reply:
Pretty sure each env variable that's added is only present within that target invocation, so this would look cleaner, but I don't think it would work.

Example:
```shell
$ cat Makefile
all: a

b: export B_VAR="b-var-value"
b:
        @echo "In 'b' target..."
        @echo B_VAR=$(B_VAR)

a: export A_VAR="a-var-value"
a: b
        @echo "In 'a' target..."
        @echo A_VAR=$(A_VAR)
        @echo B_VAR=$(B_VAR)

$ make
In 'b' target...
B_VAR=b-var-value
In 'a' target...
A_VAR=a-var-value
B_VAR=
```